src/components/domain/RiskImminentEvents/Pdc/index.tsx

```diff
@@ -59,28 +59,76 @@ function Pdc(props: Props) {
         },
     });
 
-    const { trigger: getFootprint } = useRiskLazyRequest<'/api/v1/pdc/{id}/exposure/', {
-        successCallback: FootprintCallback,
+    const {
+        response: exposureResponse,
+        pending: exposureResponsePending,
+        trigger: getFootprint,
+    } = useRiskLazyRequest<'/api/v1/pdc/{id}/exposure/', {
         eventId: number | string,
     }>({
         apiType: 'risk',
         url: '/api/v1/pdc/{id}/exposure/',
         pathVariables: ({ eventId }) => ({ id: Number(eventId) }),
-        onSuccess: (response, { successCallback }) => {
+    });
+
+    const pointFeatureSelector = useCallback(
+        (event: EventItem): EventPointFeature | undefined => {
+            const {
+                id,
+                latitude,
+                longitude,
+                hazard_type,
+            } = event;
+
+            if (isNotDefined(latitude)
+                || isNotDefined(longitude)
+                || isNotDefined(hazard_type)
+                || hazard_type === ''
```

Comment:
Let's use isFalsyString

Reply:
This is done to fix the typing, using `isFalsyString` would not solve the issue here